CMakeLists.txt

```diff
@@ -114,10 +115,21 @@ set(FREETYPE_GL_SRC
     vertex-buffer.c
 )
 
-add_library(freetype-gl STATIC
```

Comment:
If you need a shared variant drop the `STATIC` keyword instead and use `BUILD_SHARED_LIBS` as cmake parameter but don't introduce multiple targets.